core/src/banking_stage/scheduler_messages.rs

```diff
@@ -14,6 +14,12 @@ impl TransactionBatchId {
     }
 }
 
+impl Display for TransactionBatchId {
```

Comment:
TIL Display, just for my benefit, how do you decide when to impl Display over Debug?

Reply:
I use `Display` if I want a manual implementation for the formatting; `Debug` if I just want something quick.
For something like a simple u64 wrapper, the derived `Debug` impl would show something like: `TransactionBatchId { 123 }`, which is more verbose than I reall want here.